### DISCUSSION TITLE
What does isolatesCount do?

### DISCUSSION THREAD

**Discussion by spauldhaliwal** [dsrenesanse/worker_manager#82] [Q&A] [answered]
If I pass say, 5, to isolatesCount, then it will make 5 isolates. Does the plugin automatically assign new calls to each isolates and spread the load? Or do I have to explicitly tell executor somehow which function calls I want to happen on which isolate?

Are there any recommendations on when to use more than one isolate? I'm assuming the default is 1?

Thanks

**Comment by dsrenesanse** [comment, marked as answer]
you dont need to specify exact isolate to run your function, also executor is automatically detecting how many processors available and schedule task across of them, you can pass your custom value sometimes because it could be useful on very specific and cheap android devices.
